### PR TITLE
To-do-List Tutorial

### PR DESCRIPTION
**Pull request by Patfarmurs**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    monospace;
+}
+
+.todo {
+  cursor: pointer;
+}
+
+.strike {
+  text-decoration: line-through;
+}
```
Comment (AmineDerbal) [inline src/index.css:35]: - Kindly add some styling to your Application

**Review by AmineDerbal — CHANGES_REQUESTED** [review]
Hi @Patfarmurs , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️

- no linters ❌ 
- no Readme ❌ 

_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Patfarmurs** [comment]
Thank you @AmineDerbal.

**Review by AmineDerbal — APPROVED** [review]
**Hi @Patfarmurs 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Professional readme file
✔️ Clean Codebase
✔️ Linters

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Comment by Patfarmurs** [comment]
Thank you @AmineDerbal

**Pull request merged**